app/models/feedback.rb

```diff
@@ -52,20 +53,23 @@ def description
 
   def save_feedback
     response = SurveyMonkeySender.call(self)
-    @response_failed_message = "Unable to submit feedback [#{response[:error_code]}]" unless response[:success]
+    if response[:success]
+      @response_succeeded_message = 'Feedback submitted'
+    else
+      @response_failed_message = "Unable to submit feedback [#{response[:error_code]}]"
+    end
     response[:success]
   end
 
   def save_bug_report
-    ZendeskSender.send!(self) unless is_feedback_with_empty_comment?
-    true
+    ZendeskSender.send!(self)
+    @response_succeeded_message = 'Fault reported'
+  rescue ZendeskAPI::Error::ClientError => e
+    @response_failed_message = "Unable to submit fault report [#{e}]"
```

Comment:
:+1: for giving the user feedback when there is a submission failure.

We probably do not want to be putting the `ZendeskAPI::Error::ClientError` directly in here as we do not have control over what information is included. I got this;

![Screenshot 2021-10-06 at 15 18 40](https://user-images.githubusercontent.com/4415912/136221447-06901ffe-263f-4f8c-9307-76e66aebd255.png)

which gives unnecessary information to the user. It would be helpful to have an error code, like Survey Monkey, but I don't see that anywhere.

Reply:
yeah, we discussed this in tuple session. happy to remove but perhaps log the detail instead (as sentry will not the error detail up)?